### PR TITLE
strigify internal object type value

### PR DESCRIPTION
**Pull request by ros3bud**
Solves #24 

@cgrayson, LGTY?

### CODE REVIEW

File: src/common.coffee [hunk 1]
```diff
@@ -18,10 +25,6 @@ createRequest = (vars, profile) ->
       'Content-Type': 'application/x-www-form-urlencoded'
       'X-API-KEY': vars.apiKey
 
-
-  vars.apiKey = Array(vars.apiKey.length + 1).join('*')
```
Comment (cgrayson) [inline src/common.coffee]: Is this no longer needed?

File: src/common.coffee [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
   body =
     emails: vars.lead.email
     profile: profile
 
-  body = querystring.encode(body)
+  for name, value of body
+    if value
+      body[name] = value.toString()
+    else
+      delete body[name]
```
Comment (cgrayson) [inline src/common.coffee]: It looks like this loop is traversing over an object that always has exactly two elements, defined immediately prior. Why not have the `valueOf()` (preferred over `toString()`) in the assignment of the 2 attributes of `body`?

File: src/common.coffee (hunk 1 above)
Comment (ros3bud) [inline src/common.coffee]: It still is, thanks for catching. I took it out when testing locally.

File: src/common.coffee (hunk 2 above)
Comment (ros3bud) [inline src/common.coffee]: My thinking on this was that adding this method here allows us to standardize how this is done regardless of how many elements are there. But I see what you mean - it seems like a lot to do for only 2 elements - seems like the number of elements should always be considered before adding this method.

**Comment by cgrayson** [comment]
:+1:

**Pull request merged**